website/source/docs/internals/acl.html.markdown

```diff
@@ -86,9 +86,13 @@ key "_rexec/" {
 
 ### Bootstrapping ACLs
 
-Bootstrapping the ACL system is done by providing an initial `acl_master_token`
-[configuration](/docs/agent/options.html), which will be created as a
-"management" type token if it does not exist.
+Bootstrapping the ACL system is done by providing an initial [`acl_master_token`
```

Comment:
There's some ambiguity here between "`acl_master_token`" and "`acl_master` token" :smile: